css/themes/_light.scss

```diff
@@ -91,7 +91,17 @@ $popoverFontColor: #ffffff;
 $popupMenuSelectedItemBackground: rgba(256, 256, 256, .2);
 
 // Toolbar
+$secToolbarBackground: rgba(0, 0, 0, 0.5);
+$sideBarBackground: rgba(0, 0, 0, 0.75);
 $splitterColor: #ccc;
+$toolbarBackground: rgba(0, 0, 0, 0.5);
+$toolbarBadgeBackground: #165ECC;
+$toolbarBadgeColor: #FFFFFF;
+$toolbarButtonColor: #FFFFFF;
+$toolbarSelectBackground: rgba(0, 0, 0, .6);
+$toolbarTitleColor: #FFFFFF;
+$toolbarToggleBackground: #12499C;
```

Comment:
I don't fully understand the distinction between variables.scss and a themes scss file. It seems like both have variables and both are necessary for scss to compile. What decides when a variables goes into one or the other?